ghost/core/test/e2e-api/shared/__snapshots__/version.test.js.snap

```diff
@@ -813,7 +813,7 @@ generated some helpful information about the error that you can share with the
 creator of the Zapier integration below: 
 
  Integration expected Ghost version:v3.1 
-Current Ghost version:v5.30 
```

Comment:
This test should not be using the email snapshots just yet as there's a dynamic bit with Ghost version. Which is a [documented limitation](https://ghost.notion.site/End-to-end-Testing-6a2ef073b1754b18aff42e24a632a007#3a22bcdf5ab747309d8ab1bef8b98578) atm.